src/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 600;
+  font-style: normal;
+}
+
+.wrapper {
+  height: 130px;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: "Avenir-Book", sans-serif;
+  width: 100%;
+}
+
+.search-big > input {
+  height: 70px;
+  padding-left: 62px;
+  line-height: 22px;
+  background: url(images/Search.svg) no-repeat 26px 25px;
+  font-size: 16px;
+}
+
+.search-small > input {
+  height: 42px;
+  padding-left: 33px;
+  background: url(images/Search.svg) no-repeat 12px 14px/11px 11px;
+  line-height: 16px;
+  font-size: 14px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
```

Comment:
`f-w` must be changed only for input text, not for placeholder.

Reply:
if not placeholder the test fails

Reply:
there is no test for placeholder `font-weight`